packages/cyberstorm/src/components/SelectItem.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/SelectItem.module.css";
+
+export interface SelectItemProps {
+  label?: string;
+  leftIcon?: ReactNode;
+  rightIcon?: ReactNode;
+  colorScheme?: "default" | "defaultDark" | "primary";
+}
+
+/**
+ * Cyberstorm SelectItem component
+ */
+export const SelectItem: React.FC<SelectItemProps> = React.forwardRef(
+  (props, ref) => {
+    const { label, leftIcon, rightIcon, colorScheme } = props;
+
+    return (
+      <div
+        {...props}
+        ref={ref}
+        className={`${styles.root} ${getStyle(colorScheme)}`}
+      >
+        <div className={`${styles.icon} ${getStyle(colorScheme)}`}>
```

Comment:
Are the divs rendered on purpose even if left/rightIcon are undefined?

Reply:
Fixed